### PR TITLE
CASI 4 Updates

### PR DESCRIPTION
**Pull request by derekpierre**
Based over #61 .

- [x] Update `nucypher` dependency to 2.1.0b1
- [x] Update references from `cassandra` to `gemini`
- [x] Update default teacher to use teacher for `gemini` network
- [x] Fix tooltip text that is too light and uses `configurations` instead of `confirmations` (https://github.com/nucypher/nucypher-monitor/pull/61#issuecomment-607265610) 😅

### CODE REVIEW

**Comment by derekpierre** [comment]
@KPrasch - ready other than a minor issue with the staking graph since the globally staked NU is currently 0. Presumably this will update in the next period.

Screen Shot 2020-04-02 at 11 10 27 AM

**Comment by derekpierre** [comment]
I figured out the" today line" issue, but not sure of the fix just yet. Sometimes, or at least right now when there is no historical data, today's date is not a value in the x-axis range (starts from tomorrow for future stakers). So when we try to plot the today line, it's x-axis value is just plotted at the end - hence the today line all the way to the right

I think this is something we can fix later on in a follow-up issue.

File: tests/test_crawler.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ def create_crawler(node_db_filepath: str = IN_MEMORY_FILEPATH, dont_set_teacher:
     teacher_nodes = None
     if not dont_set_teacher:
         teacher_nodes = actions.load_seednodes(None,  # TODO: Needs emitter
-                                               teacher_uris=['https://discover.nucypher.network:9151'],  # TODO: Needs Cleanup
+                                               teacher_uris=['https://gemini.nucypher.network:9151'],  # TODO: Needs Cleanup
```
Comment (KPrasch) [inline tests/test_crawler.py:208]: Does it make sense to reference `DEFAULT_TEACHER` here?

Comment (derekpierre) [reply tests/test_crawler.py]: Sure. Note however that this code doesn't get used in testing due to other blockchain testing issues.

**Pull request merged**